Could someone assist me on this syntax check as I did not modify this line:

```
2022-04-06 03:48:25 [INFO]   ---------------------------
2022-04-06 03:48:25 [INFO]   File:[/github/workspace/startup_scripts/370_ip_addresses.py]
2022-04-06 03:48:25 [ERROR]   Found errors in [mypy] linter!
2022-04-06 03:48:25 [ERROR]   Error code: 1. Command output:
------
startup_scripts/370_ip_addresses.py:54: error: Dict entry 0 has incompatible type "Optional[str]": "Any"; expected "str": "Any"
startup_scripts/370_ip_addresses.py:56: error: Item "None" of "Optional[Any]" has no attribute "objects"
Found 2 errors in 1 file (checked 1 source file)
```

--- a/startup_scripts/370_ip_addresses.py
+++ b/startup_scripts/370_ip_addresses.py
@@ -17,7 +17,7 @@ if ip_addresses is None:
 optional_assocs = {
     "tenant": (Tenant, "name"),
     "vrf": (VRF, "name"),
-    "interface": (None, None),
+    "interface": (Interface, "name"),
 }
 
 vm_interface_ct = ContentType.objects.filter(

